rust/agents/scraper/migration/src/m20220805_000003_create_table_cursor.rs

```diff
@@ -0,0 +1,51 @@
+use sea_orm_migration::prelude::*;
+
+use crate::m20220805_000001_create_table_domain::Domain;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(Cursor::Table)
```

Comment:
A cursor is for the program logic, I am predicting that we will want to store what was last scraped so on restart the scraper won't have to go from whatever the last thing it found was (it could be a long time between abacus events).